discovery-provider/plugins/network-monitoring/src/metrics/index.ts

```diff
@@ -69,6 +72,7 @@ export const generateMetrics = async (run_id: number) => {
 
   if (userCount > 0) {
     await publishSlackReport({
+      runStartTime: runStartTime.toString(),
```

Comment:
mind adding a duration field as well for readability?

Reply:
Will make a new PR with it